opengvl/mapper/gemini_mapper.py

```diff
@@ -0,0 +1,101 @@
+from opengvl.mapper.base import BaseMapper
+from google import genai
+from google.genai import types
+from dotenv import load_dotenv
+import json
+import os
+import time
+from opengvl.utils.errors import PercentagesNormalizationError
+
+load_dotenv()
```

Comment:
I would prefer to drop load_dotenv here, eventually if you want to run this as a script I would move this to if __name__ == '__main__' (...)       then load_dotenv(override=True)